_rules/SC2-1-2-no-keyboard-trap.md

```diff
@@ -0,0 +1,241 @@
+---
+name: No keyboard trap
+rule_type: composite
+description: |
+  This rule checks if it is possible to use standard keyboard navigation to navigate through all content on a web page without becoming trapped in any element, or that it is possible to use non-standard keyboard navigation to navigate through content where focus is trapped when using standard ways of keyboard navigation.
+
+success_criterion:
+- 2.1.2 # No Keyboard Trap (A)
+
+atomic_rules:
+- [No keyboard trap standard navigation](SC2-1-2-no-keyboard-trap-standard-navigation)
+- [No keyboard trap non-standard navigation](SC2-1-2-no-keyboard-trap-non-standard-navigation)
+
+authors:
+- Geir Sindre Fossøy
+- Dagfinn Rømen
+- Anne Thyme Nørregaard
+---
+
+## Aggregation Definition
+
+### Applicability
+
+The rule applies to any HTML or SVG element on a web page that is [focusable](#focusable).
+
+Note: The WCAG 2.0 success criterion 2.1.2 applies to all content, not just HTML or SVG, where focus can be moved to through keyboard navigation.
```

Comment:
What does that mean? Maybe you want to state explicitly "This rule is only applies to HTML and SVG. Thus, it is a partial checks for WCAG Success Criterion 2.1.2, which applies to all content."?

Reply:
Recommend dropping the "2.0" and just say "WCAG"

Reply:
Agree with @nitedog. The note, as it stands, makes me wonder if it is extending the applicability to non HTML or SVG elements. @nitedog suggestion is much clearer. But I'm not sure if removing 2.0 is a good idea (what if future versions of WCAG renumber the success criteria?)

Reply:
OK "WCAG 2.1" -> "WCAG 2"